### PR TITLE
MMDP-229 Fix pagination and failing tests.

### PR DESCRIPTION
**Pull request by Musacoli**
**What does this PR do?**
Fixes the pagination feature that was failing before and tests.

Summarise the main tasks handled in the pull request

**Description of Task to be completed?**

- Switch between pages of users.

**How should this be manually tested?**
- Fetch and checkout to `ft-filtering-users-cms-MMDP-229`
- Start server by running `yarn dev-sever`
- On your browser navigate to `http://localhost:1234/users/all`
- click `the next button to whatever page is available`

**Any background context you want to provide?**
None

**What are the relevant Jira issues?**

[MMDP-229](https://viisaus.atlassian.net/browse/MMDP-229)

**Screenshots (if appropriate)**
image

**Questions:**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "test:client": "jest \"-u\" --coverage",
     "test:client:watch": "jest \"-u\" --watch",
     "coverage:server": "nyc --reporter html --reporter text --report-dir ./server/coverage npm run test:server",
-    "pretest:client": "yarn lint",
```
Comment (shemogumbe) [inline package.json]: Any reason for why this was removed, remember linting is part of our tests

Comment (Musacoli) [reply package.json]: Apologies that was removed to try and make testing locally faster, it was an oversight to not put it back after.

But have made a fix to have it added.

File: server/__tests__/tests/api/resources/media.spec.js [hunk 2]
```diff
@@ -50,11 +50,6 @@ describe('Media route', () => {
       expect(res.body.data.media).toMatchObject({ media: data.video });
     });
 
-    it('should return fail when media file is missing', async () => {
```
Comment (charisschomba) [inline server/__tests__/tests/api/resources/media.spec.js]: @Musacoli Is there any reason for deleting this test? If so, clarify. Else undo the removal.

Comment (Musacoli) [reply server/__tests__/tests/api/resources/media.spec.js]: I deleted the test temporarily to be able to check coverage of the entire project as that one test was blocking, but it is to be reinstated as soon as I get the fix for it.

**Comment by charisschomba** [comment]
> @Musacoli Nice work on the implementation but fix this issues.
> 
> * Kindly follow the designs
> 
> screenshot 2019-02-28 at 14 36 25
> 
> * Fix this linting issues
> 
> screenshot 2019-02-28 at 14 40 22
> 
> * 4 tests are failing
> 
> screenshot 2019-02-28 at 14 42 55
> 
> * This file was 100% covered before your implementation, consider adding more tests
> 
> screenshot 2019-02-28 at 14 45 49
> 
> * Your implementation seems to have affected user deletion. After deleting a user the page does not update users....This is because you added a payload when fetching users `fetchingStarted action`  and you did not refactor deleteUser saga. I have a attached a fix for this.
> 
> screenshot 2019-02-28 at 14 52 41
> 
> * add a default payload for fetching users.
> 
> screenshot 2019-02-28 at 14 55 16

@Musacoli fix this issue

**Pull request merged**